pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -47,8 +57,32 @@
 @ThreadSafe
 public class BrokerReduceService {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(BrokerReduceService.class);
+
+  // brw -> Shorthand for broker reduce worker threads.
+  private static final String REDUCE_THREAD_NAME_FORMAT = "brw-%d";
+  protected static final int QUERY_RUNNER_THREAD_PRIORITY = 7;
```

Comment:
Any specific reason for this priority? Some comments will be appreciated

Reply:
This is the same as the server side code, will add comments.